README.md

```diff
@@ -48,6 +48,7 @@ Below is the list of variables you can redefine in your playbook to customize st
 | `st2_packs`              | `[ st2 ]`     | List of packs to install. This flag does not work with a `--python3` only pack.
 | `st2_python_packages`    | `[ ]`         | List of python packages to install into the `/opt/stackstorm/st2` virtualenv. This is needed when deploying alternative auth or coordination backends which depend on Python modules to make them work.
 | **st2web**
+| `st2web_version`            | `latest`      | `st2web` version to install. `present` to install available package, `latest` to get automatic updates, or pin it to numeric version like `2.2.0`.
```

Comment:
Can you please also list the `st2web_revision` var in the README? This looks like oversight from the past.